app/screens/send-bitcoin-screen/send-bitcoin-details-screen.tsx

```diff
@@ -381,6 +381,15 @@ const SendBitcoinDetailsScreen: React.FC<Props> = ({ route }) => {
           }
 
           const result = await requestInvoice(requestInvoiceParams)
+
+          setPaymentDetail((prev) => {
+            if (!prev || !prev.setSuccessAction) {
+              return prev
+            }
+
+            return prev.setSuccessAction(result.successAction)
+          })
```

Comment:
early return is good but try to avoid negated if statements when is not needed (i.e try to follow "positive condition principle")

```suggestion
          setPaymentDetail((prev) => {
            if (prev?.setSuccessAction) {
              return prev.setSuccessAction(result.successAction)
            }
 
            return prev
          })
```